src/view/styles.js

```diff
@@ -0,0 +1,490 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module engine/view/styles
+ */
+
+import { get, isObject, merge, set, unset } from 'lodash-es';
+import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import mix from '@ckeditor/ckeditor5-utils/src/mix';
+
+import BorderStyles from './styles/borderstyles';
```

Comment:
Also, I'd do that via `editor.editing.view.document.addStyleProcessor()` which will be a shorthand to call `StylesMap.addProcessor()`. This way, if we'll ever decide to bind the processor with a particular document instance, it will not change the API.

Reply:
(☝️ I used a new naming above – see next comments)